packaging/windows/khiops_env.cmd

```diff
@@ -0,0 +1,158 @@
+:DISPLAY_ENV
+setlocal
+set DISPLAY_ENV=true
+
+:SET_ENV
+REM Initialize exported variables
+set "KHIOPS_PATH="
+set "KHIOPS_COCLUSTERING_PATH="
+set "KHIOPS_MPI_COMMAND="
+set "KHIOPS_JAVA_PATH="
+set "KHIOPS_CLASSPATH="
+set "KHIOPS_JAVA_ERROR="
+set "KHIOPS_MPI_ERROR="
+
+REM Set Khiops home to parent directory
+for %%a in ("%~dp0..") do set "_KHIOPS_HOME=%%~fa"
+
+REM KHIOPS_PATH
+set "KHIOPS_PATH=%_KHIOPS_HOME%\bin\MODL.exe"
+set "KHIOPS_COCLUSTERING_PATH=%_KHIOPS_HOME%\bin\MODL_Coclustering.exe"
+
+REM KHIOPS_LAST_RUN_DIR
+if "%KHIOPS_LAST_RUN_DIR%".=="". set "KHIOPS_LAST_RUN_DIR=%USERPROFILE%\khiops_data\lastrun"
+
+REM KHIOPS_PROC_NUMBER
+if "%KHIOPS_PROC_NUMBER%".=="". for /f %%i in ('"%~dp0_khiopsgetprocnumber"') do set "KHIOPS_PROC_NUMBER=%%i"
+if "%KHIOPS_PROC_NUMBER%".=="". set "KHIOPS_PROC_NUMBER=1"
+
+REM set MPI environment
+if %KHIOPS_PROC_NUMBER% LEQ 2 goto MPI_DONE
+REM ... with the standard variable MSMPI_BIN
+set "KHIOPS_MPI_COMMAND=%MSMPI_BIN%mpiexec.exe"
+REM ... if MSMPI_BIN is not correctly defined 
+REM ... we try to call directly mpiexec (assuming its path is in the 'path' variable)
+if not exist "%KHIOPS_MPI_COMMAND%" goto USE_MPI_IN_PATH
+:MPI_PARAMS
+REM ... we add the MPI parameters
+if not "%KHIOPS_MPI_COMMAND%."=="." set "KHIOPS_MPI_COMMAND="%KHIOPS_MPI_COMMAND%" -n %KHIOPS_PROC_NUMBER%"
+:MPI_DONE
+
+set _KHIOPS_GUI=@GUI_STATUS@
+if "%_KHIOPS_GUI%" == "false" GOTO SKIP_GUI
+
+REM Set Java environment
+set _JAVA_ERROR=false
+if not exist "%_KHIOPS_HOME%\jre\bin\server\" set _JAVA_ERROR=true
+if not exist "%_KHIOPS_HOME%\jre\bin\" set _JAVA_ERROR=true
+
+if  "%_JAVA_ERROR%" == "false" (
+    set "KHIOPS_JAVA_PATH=%_KHIOPS_HOME%\jre\bin\server\;%_KHIOPS_HOME%\jre\bin\"
+) else set "KHIOPS_JAVA_ERROR=The JRE is missing in Khiops home directory, please reinstall Khiops"
+
+REM KHIOPS_CLASSPATH
+set "KHIOPS_CLASSPATH=%_KHIOPS_HOME%\bin\norm.jar"
+set "KHIOPS_CLASSPATH=%_KHIOPS_HOME%\bin\khiops.jar;%KHIOPS_CLASSPATH%"
+
+:SKIP_GUI
+
+REM unset local variables
+set "_KHIOPS_GUI="
+set "_JAVA_ERROR="
+set "_KHIOPS_HOME="
+
+if not "%DISPLAY_ENV%" == "true" exit /b 0
+
+REM Print the environment list on the standard output
+echo KHIOPS_PATH %KHIOPS_PATH%
+echo KHIOPS_COCLUSTERING_PATH %KHIOPS_COCLUSTERING_PATH%
+echo KHIOPS_MPI_COMMAND %KHIOPS_MPI_COMMAND%
+echo KHIOPS_JAVA_PATH %KHIOPS_JAVA_PATH%
+echo KHIOPS_CLASSPATH %KHIOPS_CLASSPATH%
+echo KHIOPS_LAST_RUN_DIR %KHIOPS_LAST_RUN_DIR%
+echo KHIOPS_PROC_NUMBER %KHIOPS_PROC_NUMBER%
+echo KHIOPS_TMP_DIR %KHIOPS_TMP_DIR%
+echo KHIOPS_MEMORY_LIMIT %KHIOPS_MEMORY_LIMIT%
+echo KHIOPS_API_MODE %KHIOPS_API_MODE%
+echo KHIOPS_RAW_GUI %KHIOPS_RAW_GUI%
+echo KHIOPS_JAVA_ERROR %KHIOPS_JAVA_ERROR%
+echo KHIOPS_MPI_ERROR %KHIOPS_MPI_ERROR%
+endlocal
+exit /b 0
+
+:USE_MPI_IN_PATH
+REM the standard variable MSMPI_BIN is not correctly defined
+REM ... we try to call directly mpiexec (assuming its path is in the 'path' variable)
+set "KHIOPS_MPI_COMMAND=mpiexec"
```

Comment:
IMHO, in Conda environments we shouldn't rely on finding `mpiexec` in the `PATH`, for the same reasons as stated in the comment on the same issue for the `khiops_env` UNIX script. Like in that comment, I would go for an equivalent solution of something like:

```cmd
set "KHIOPS_MPI_COMMAND=%~dp0mpiexec"
```